custom_conf.py

```diff
@@ -97,6 +97,9 @@
 
     # Controls the existence of Previous / Next buttons at the bottom of pages
     # Valid options: none, prev, next, both
+    # You can override the default setting on a page-by-page basis by specifying
+    # it as file-wide metadata at the top of the file, see
+    # https://www.sphinx-doc.org/en/master/usage/restructuredtext/field-lists.html
```

Comment:
the upstream doc page doesn't help. can you provide a simple example in these comments (or update upstream)?

Reply:
Why does it not help? The syntax for RST is `:fieldname: Field content`, which is the first example on the page.
However, the syntax is different for MyST, which is why I'm linking to the page to give some context, so users know what to search for when they want to do this in MyST.

Of course I could include examples here, but then we'll need to cover both RST and MyST, and I feel the configuration file is the wrong place for this.
Ideally, this should go into the readme imo, but as far as I remember you feel the readme is way too long as it is?

Reply:
What **exactly** is placed at the top of a page and how is it used to influence sequence navigation icons?

If the content placeholder "Field content" applies to the entire page, does it have any real effect?

I don't mind adding information to the repository documentation. My earlier comments were made within the context of having everything on a single page.

Reply:
> What **exactly** is placed at the top of a page and how is it used to influence sequence navigation icons?

`:sequential_nav: none`, `:sequential_nav: prev`, `:sequential_nav: next`, or `:sequential_nav: both` for RST or

```
---
sequential_nav: none
---
```

```
---
sequential_nav: prev
---
```

```
---
sequential_nav: next
---
```

or 
```
---
sequential_nav: both
---
``` 

for MyST.

> If the content placeholder "Field content" applies to the entire page, does it have any real effect?

Yes, if you do not want the navigation buttons in your documentation in general, but you want them for a tutorial, you can enable them in the tutorial pages. That was a request that just came up.

> I don't mind adding information to the repository documentation. My earlier comments were made within the context of having everything on a single page.

I can add a section about sequential navigation to the readme, no problem.
But I'm not convinced at all that it is a good idea to split the readme into a bunch of separate pages - from a documentation perspective it makes sense of course, but the starter pack is not a set of documentation, it's a tool. And the more documentation files we create, the harder it is to actually start using the starter pack, because you need to figure out which files you actually need and which you should delete. Plus, you cannot just open the readme and search for how to do something, but you need to go to the doc output to find out.

Reply:
Let's merge this current PR now. Followed by PR #165 . Then a PR documenting the above.